digi/xbee/reader.py

```diff
@@ -616,6 +666,30 @@ def del_user_data_relay_received_callback(self, callback):
         """
         self.__relay_data_received += callback
```

Comment:
Not from this commit, but this should be `self.__relay_data_received -= callback`